app/src/componentLibrary/sidebar/sidebarButton/sidebarButton.jsx

```diff
@@ -16,35 +16,31 @@
 
 import { PropTypes } from 'prop-types';
 import classNames from 'classnames/bind';
+import { NavLink } from 'components/main/navLink';
+import Parser from 'html-react-parser';
 import styles from './sidebarButton.scss';
 
 const cx = classNames.bind(styles);
 
-export const SidebarButton = ({
-  className,
-  onClick,
-  onMouseEnter,
-  onMouseLeave,
-  onMouseDown,
-  children,
-}) => (
-  <button
-    className={cx('sidebar-button', className)}
-    onClick={onClick}
-    tabIndex={-1}
-    onMouseEnter={onMouseEnter}
-    onMouseLeave={onMouseLeave}
-    onMouseDown={onMouseDown}
+export const SidebarButton = ({ icon, onClick, message, onLeaveSidebar, link }) => (
+  <NavLink
+    to={link}
+    className={cx('sidebar-button')}
+    activeClassName={cx('active')}
+    onClick={() => {
+      onClick();
+      onLeaveSidebar();
```

Comment:
Can it be managed at the top level and used here via single `onClick` prop?

Reply:
Done.